### ISSUE TITLE
Add search button to quickly look through content

### ISSUE THREAD

**Issue by chauhankaranraj** [chauhankaranraj/lit-logits#3]
SSIA
